### ISSUE TITLE
There is no focus styling on the radio boxes pattern

### ISSUE THREAD

**Issue by anthonydillon** [canonical-web-and-design/ubuntu.com#11861]
## Summary
The is currently no styling to show which card section is highlighted via keyboard. Once focused the arrow keys work well but its difficult to know where you are.

## Process
1. Go to https://ubuntu.com/advantage/subscribe
2. Click the H1
3. Tab with the keyboard down the wizard and see the focus is missing on card sections. 

## Current and expected result
I would expect to see some indication of focus.

## Screenshot
![Peek 2022-07-20 15-43](https://user-images.githubusercontent.com/1413534/180011243-f57fd7ee-fe0a-49eb-80d0-55d502f87632.gif)
